src/addon/file_server/mod.rs

```diff
@@ -282,8 +315,7 @@ impl<'a> FileServer {
         Ok(DirectoryIndex {
             entries: directory_entries,
             breadcrumbs,
-            sort_by_name: false,
-            sort_by_size: false,
+            sort: Sort::Default,
```

Comment:
Whats the `Default` sorting for?
I think in would be better to provide an explicit value like:

`Sort::Name`

Reply:
The `Default` sorting puts the directories above the files. I don't know how it works

Reply:
And since it's an enum, it must have a value

Reply:
Perhaps lets use `Directories`?